client/scripts/models/IChainAdapter.ts

```diff
@@ -62,6 +63,12 @@ abstract class IChainAdapter<C extends Coin, A extends Account<C>> {
     await this.app.reactions.refreshAll(this.id, null, true);
     await this.app.tags.refreshAll(this.id, null, true);
 
+    await $.get(`${this.app.serverUrl()}/bulkMembers`, { chain: this.id, })
+      .then((res) => {
+        const roles = res.result.filter((r) => { return r.permission === 'admin' || r.permission === 'moderator'; });
```

Comment:
Same as above

Reply:
It feels like this fetch code should be factored out as its own method on the controller, as it doesn't really belong here